src/cryptography/x509/extensions.py

```diff
@@ -342,6 +342,9 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def __hash__(self):
+        return hash(self.ca) ^ hash(self.path_length)
```

Comment:
Seems like hashing the tuple and letting the underlying implementation xor is reasonable. I'll switch it.